src/core/common/inviwocore.cpp

```diff
@@ -270,24 +270,6 @@ InviwoCore::InviwoCore(InviwoApplication* app)
     registerMetaData(std::make_unique<Size2MetaData>());
     registerMetaData(std::make_unique<Size3MetaData>());
     registerMetaData(std::make_unique<Size4MetaData>());
-    registerMetaData(std::make_unique<VectorMetaData<2, float>>());
```

Comment:
what happened to these?

Reply:
never used since we have FloatVec2MetaData

Reply:
Think I must have been young and confused when I added those may years ago ;)